packages/web-app/.storybook/en.json

```diff
@@ -1010,8 +1012,13 @@
   "theme": "theme",
   "Theme": "Theme",
   "There is no caves related to this group currently": "There is no caves related to this group currently.",
+  "There is currently no location for this entrance": "There is currently no location for this entrance.",
+  "There is currently no snapshot registered for this {type}": "There is currently no snapshot registered for this {type}.",
   "Therefore, for a few weeks, you will have the opportunity to respond to our <fundRaisingOperationUrl>fund-raising operation</fundRaisingOperationUrl>.": "Therefore, for a few weeks, you will have the opportunity to respond to our <fundRaisingOperationUrl>fund-raising operation</fundRaisingOperationUrl>.",
   "This {0} version of the site is improving gradually to allow a simpler navigation from all your peripherals": "This {0} version of the site is improving gradually to allow a simpler navigation from all your peripherals.",
+  "This entrance has a restricted access, you can not see its locations": "This entrance has a restricted access, you can not see its locations.",
+  "This {type} has no related snapshot": "This {type} has no related snapshot.",
+  "This {type} is related to a sensitive entrance": "This {type} is related to a sensitive entrance. Only administrator can see this.",
```

Comment:
si possible la clé = le texte pour l'anglais

Reply:
Same response as before, we will change this